lib/smartcar/base.rb

```diff
@@ -13,8 +13,6 @@ class Base
     class InvalidParameterValue < StandardError; end
     # Constant for Basic auth type
     BASIC = 'Basic'
-    # Number of seconds to wait for response
-    REQUEST_TIMEOUT = 310
```

Comment:
I moved this up to the `Smartcar` namespace mostly just to make it a little cleaner to access from outside the gem (as in the documentation examples), plus it probably makes sense for it to exist within the same namespace as the default `API_ORIGIN` -- but open to feedback on this. 🤷‍♂️

Reply:
Makes sense to me 👍🏽